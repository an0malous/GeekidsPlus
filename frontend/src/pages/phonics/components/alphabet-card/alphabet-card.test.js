import React from 'react'
import { shallow } from 'enzyme'
import AlphabetCard from './alphabet-card.component';

describe('AlphabetCard Component', () => {
   let component;

   beforeEach(() => {
     component = shallow(<AlphabetCard />)
   });
   
   it('Renders', () => {
      expect(component).toMatchSnapshot()
   })
});


