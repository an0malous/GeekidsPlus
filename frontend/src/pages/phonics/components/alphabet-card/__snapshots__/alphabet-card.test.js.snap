// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlphabetCard Component Renders 1`] = `
<div
  style={
    Object {
      "backgroundColor": "white",
      "border": "1px solid black",
      "borderRadius": "15%",
      "color": "red",
      "padding": "20px",
    }
  }
/>
`;
